.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!--
```

Comment:
Maybe this file needs to be referenced somewhere? Otherwise I don't see it being used. In general though, I don't know whether it is needed. All of the points listed here are listed in CONTRIBUTING.md

Reply:
This file will be used to autopopulate a PR message in the Github UI. We can of course not use an automatic template for that, but given the reality of human beings probably not first reading CONTRIBUTING.md, I thought it would be helpful to have a quick tickable list to nudge people into doing the things we like to see before a PR is merged.

Reply:
I can't reply directly to the open comment above, but it's in the commit I'm about to push.